src/DvfHelpers.php

```diff
@@ -50,8 +48,7 @@ public function transformMachineName($string) {
    *   The link.
    */
   public function getHelpPageLink($template_name) {
-    $link = Link::fromTextAndUrl('Help', Url::fromUserInput($this->helpPageBasePath . $template_name));
-    return new FormattableMarkup('<span class="dvf-admin-popup">' . $link->toString() . ' &#x29c9;</span>', []);
+    return new FormattableMarkup('<span class="dvf-admin-popup"><a href="/dvf/help/' . $template_name . '">Help</a> &#x29c9;</span>', []);
```

Comment:
Is it possible to add template variables ? Not a biggy
```suggestion
    return new FormattableMarkup('<span class="dvf-admin-popup"><a href="/dvf/help/@template_name'">Help</a> &#x29c9;</span>', ['@template_name' => $template_name]);
```